packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -84,7 +85,7 @@ export function Modal(props: PropsWithChildren<Props>) {
               typeof title !== 'string' && title
             }
             <div className={styles.modalHeaderClose}>
-              <IconButton name="times" size="xl" onClick={onDismiss} tooltip="Close" />
+              <IconButton name="times" size="xl" onClick={onDismiss} tooltip={t('modal.close-tooltip', 'Close')} />
```

Comment:
this should be in our documentation, but can you please change this ID to `t('grafana-ui.modal.close-tooltip', 'Close')` and re-run `yarn i18n:extract`?

Reply:
done. thanks